test/clj_kondo/main_test.clj

```diff
@@ -2529,6 +2529,36 @@ foo/foo ;; this does use the private var
 
 " {:linters {:unsorted-required-namespaces {:level :warning}}})))))
 
+(deftest require-prefix-form-containing-periods-test
```

Comment:
Should I add these tests into `corpus/prefixed_libspec.clj` instead?

Reply:
Then they'll be tested under https://github.com/clj-kondo/clj-kondo/blob/master/test/clj_kondo/main_test.clj#L487

Reply:
Not entirely sure that's necessary however, it might be simpler to keep them separate.

Reply:
Splitting it out either way is fine with me.